.travis.yml

```diff
@@ -7,9 +7,9 @@ before_install:
     - pip install --upgrade pip
 install:
     - make build
+    - pip install -r test_requirements.txt
     - pip install codecov
 script:
-    - make test
-    - coverage run -m unittest tests/*.py
+    - pytest tests/ --cov=app --cov-report html
```

Comment:
`pytest -v tests/ --cov=app --cov-report html` will produce a more verbose output of individual files if thats the direction we want to move in. http://pythontesting.net/framework/pytest/pytest-introduction/